index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Alzubair Protfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="container">
+    <h1 class="navbar">My Logo</h1>
+    <i class="fa-solid fa-bars"></i>
```

Comment:
- [ ] please add the hamburger icon in `nav tag`.

Reply:
hi dear @tufoinnkuo10 
 the <i class="fa-solid fa-bars"></i>  its a hamburger icon, i used the font awesome  website to get it
i think its really okay with it

Reply:
yes please put it inside a `nav tag`